index.html

```diff
@@ -1,63 +1,233 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="style.css" />
-    <link rel="preconnect" href="https://fonts.gstatic.com" />
-    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap" rel="stylesheet" />
-    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap" rel="stylesheet" />
-    <link href="https://unpkg.com/boxicons@2.0.7/css/boxicons.min.css" rel="stylesheet" />
-    <link
-      rel="stylesheet"
-      href="https://use.fontawesome.com/releases/v5.15.3/css/all.css"
-      integrity="sha384-SZXxX4whJ79/gErwcOYf+zWLeJdY/qpuqC4cAa9rOGUstPomtqpuNWT9wdPEn2fk"
-      crossorigin="anonymous"
-    />
-          <title>Lovi Madaan Portfolio</title>
-  </head>
-  <body>
-    <main>
-      <div class="main-content">
-        <!-- header and Navigation -->
-        <header>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="style.css" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" />
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap" rel="stylesheet" />
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap" rel="stylesheet" />
+  <link href="https://unpkg.com/boxicons@2.0.7/css/boxicons.min.css" rel="stylesheet" />
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.3/css/all.css"
+    integrity="sha384-SZXxX4whJ79/gErwcOYf+zWLeJdY/qpuqC4cAa9rOGUstPomtqpuNWT9wdPEn2fk" crossorigin="anonymous" />
+  <title>Lovi Madaan Portfolio</title>
+</head>
+
+<body>
+  <main>
+    <div class="main-content">
+      <!-- header and Navigation -->
+      <header>
+        <a href="#">
+          <h1 class="logo">LM</h1>
+        </a>
+        <nav>
           <a href="#">
-            <h1 class="logo">LM</h1>
+            <i class="fas fa-bars menu hamburger-menu" id="hamburger"></i>
```

Comment:
- [ ] Hi @LoviMadaan, great effort working on the mobile menu of your portfolio project 👍 👍, I would kindly suggest we follow the project requirement as much as possible, According to the requirement it was stated that when a user clicks on any option of the mobile menu link, a part of the link in the page should be displayed, but I notice the functionality is not implemented in your project, kindly fix it.

Project requirement 👇 

![annie](https://user-images.githubusercontent.com/78634841/217300463-40575094-ac27-42a4-aba0-8fe8da865c4d.png)